cmd/main.go

```diff
@@ -63,25 +63,24 @@ func main() {
 	}
 
 	// Neeed at least two interfaces
-	total := len(promisc) + len(listen)
-	if total < 2 {
-		log.Fatal("Please specify --promisc and --listen at least twice in total")
+	if len(interfaces) < 2 {
+		log.Fatal("Please specify --interfaces at least twice")
 	}
 
 	// handle our timeout & bpf filter for ports
 	to := parseTimeout(timeout)
 	bpf_filter := buildBPFFilter(ports)
 
-	// init the listeners which are not promisc
-	listeners := initalizeListeners(listen, false, bpf_filter, ports, to)
+	// init the listeners
+	listeners := initializeListeners(interfaces, bpf_filter, ports, to)
 	// do the same for the promisc list which is
-	for _, l := range initalizeListeners(promisc, true, bpf_filter, ports, to) {
-		listeners = append(listeners, l)
-	}
+	//for _, l := range initializeListeners(promisc, true, bpf_filter, ports, to) {
+	//	listeners = append(listeners, l)
```

Comment:
i realize I didn't explicitly point this one out, but I was hoping you'd clean it up.

Reply:
yep... forgot to delete those